db/migrate/20200517204505_create_meetings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMeetings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :meetings do |t|
+      t.string :name
+      t.datetime :start_time
+      t.datetime :end_time
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
What is the use of this user_id
How are we managing multiple interview participants in an interview?